openapi.yaml

```diff
@@ -26,7 +28,552 @@ tags:
+      responses:
+        '200':
+          description: Array of registered platforms
+  /api/compute/platforms/{platformId}/configurations:
+    get:
+      tags:
+        - admins
+        - researchers
+        - Compute
+      summary: Get configuration available for a specific platform. As an example this can be EC2 instance types available for EC2 Linux
+      operationId: GetConfigForPlatform
+      parameters:
+        - name: platformId
+          in: path
+          required: true
+          schema:
+            $ref: '#/components/schemas/PlatformId'
+      description: |
+        Get configuration available for a specific platform
+      responses:
+        '200':
+          description: Platform Configuration JSON
+  /api/aws-accounts:
+    get:
+      tags:
+        - admins
+        - AwsAccounts
+      summary: List all registered AWS accounts
+      operationId: ListAwsAccounts
+      description: |
+        List all registered AWS accounts
+      responses:
+        '200':
+          description: Array of registered AWS accounts
+    post:
+      tags:
+        - admins
+        - AwsAccounts
+      summary: Create a new AWS Account
+      operationId: CreateAwsAccount
+      description: |
+        Create the AWS Account. Note that this doesn't provision a new account, just adds an entry in backend
+      requestBody:
+        description: AWS Account information
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateAwsAccountRequest'
+      responses:
+        '200':
+          description: AWS Account information added
+  /api/aws-accounts/{awsAccountUUID}:
+    get:
+      tags:
+        - admins
+        - AwsAccounts
+      summary: Get AWS account information
+      operationId: GetAwsAccount
+      parameters:
+        - name: awsAccountUUID
+          in: path
+          required: true
+          schema:
+            $ref: '#/components/schemas/AwsAccountUUID'
+      description: |
+        Get AWS Account information
+      responses:
+        '200':
+          description: AWS Account information
+  /api/aws-accounts/provision:
+    post:
+      tags:
+        - admins
+        - AwsAccounts
+      summary: Provision a new AWS Account
+      operationId: ProvisionAwsAccounts
+      description: |
+        Provision a new AWS Account
+      requestBody:
+        description: AWS Account information
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ProvisionAccountRequest'
+      responses:
+        '200':
+          description: AWS Account provisioning started
+  /api/accounts:
+    get:
+      tags:
+        - admins
+        - Accounts
+      summary: List all registered accounts
+      operationId: ListAccounts
+      description: |
+        List all registered accounts
+      responses:
+        '200':
+          description: Array of registered accounts
+    post:
```

Comment:
What's the difference between this one and `/aws-accounts/provision` ?

Reply:
From what I could see there are two registered APIs and they do the same thing pretty much

Reply:
Yes, logging it in a tech debt ticket would be great!